executor/stale_txn_test.go

```diff
@@ -733,6 +733,11 @@ func (s *testStaleTxnSuite) TestAsOfTimestampCompatibility(c *C) {
 		c.Assert(err.Error(), Matches, ".*as of timestamp can't be set in transaction.*")
 		tk.MustExec("commit")
 	}
+	_, err := tk.Exec(`create table test.table1 (id int primary key, a int);`)
+	c.Assert(err, IsNil)
+	time1 = time.Now()
+	_, err = tk.Exec(fmt.Sprintf("explain analyze select * from t5 as of timestamp '%s' where id = 1;", time1.Format("2006-1-2 15:04:05.000")))
```

Comment:
You could just use `MustExec` and `CURRENT_TIMESTAMP(6)` instead of the additional format.

Reply:
updated.

Reply:
May we also add the corresponding `drop` statement?